java/src/jmri/jmrix/SystemsMenu.java

```diff
@@ -43,61 +45,73 @@ public SystemsMenu() {
                 add(menu);
             }
         }
+
         add(new javax.swing.JSeparator());
 
-        addMenu("jmri.jmrix.acela.AcelaMenu");
-        addMenu("jmri.jmrix.bachrus.SpeedoMenu");
+        // Acela is migrated
+        add(new jmri.jmrix.acela.AcelaMenu(null));
+        // Bachrus is migrated
+        add(new jmri.jmrix.bachrus.SpeedoMenu(null));
         // CAN is migrated
         add(new jmri.jmrix.can.swing.CanMenu(null));
-
         // Merg CBUS is migrated
         add(new jmri.jmrix.can.cbus.swing.CbusMenu(null));
-
-        addMenu("jmri.jmrix.cmri.CMRIMenu");
+        // C/MRI is migrated
+        add(new jmri.jmrix.cmri.CMRIMenu(null));
+        // DCC++ is migrated
         add(new jmri.jmrix.dccpp.swing.DCCppMenu(null));
-        addMenu("jmri.jmrix.easydcc.EasyDCCMenu");
+        // EasyDCC is migrated
+        add(new jmri.jmrix.easydcc.EasyDccMenu(null));
+
         addMenu("jmri.jmrix.grapevine.GrapevineMenu");
 
         // LocoNet is migrated
         add(new jmri.jmrix.loconet.swing.LocoNetMenu(null));
-        //addMenu("jmri.jmrix.loconet.swing.LocoNetMenu");
         // NCE is migrated
         add(new jmri.jmrix.nce.swing.NceMenu(null));
-
         // OpenLCB is migrated
         add(new jmri.jmrix.openlcb.OpenLcbMenu(null));
 
         addMenu("jmri.jmrix.oaktree.OakTreeMenu");
+
         // Powerline is migrated
         add(new jmri.jmrix.powerline.swing.PowerlineMenu(null));
-        addMenu("jmri.jmrix.pricom.PricomMenu");
-        addMenu("jmri.jmrix.qsi.QSIMenu");
+        addMenu("jmri.jmrix.pricom.PricomMenu"); // special type of connection, not to be migrated
+        // QSI is migrated
+        add(new jmri.jmrix.qsi.QSIMenu(null));
```

Comment:
This is the line causing the findbugs error, but the actual issue is in the QSIMenu class, which has an annotation that says the single parameter cannot be null.

Reply:
The jmrix.SystemsMenu package has been removed. @pabender time permitting, can you sign off the other items? I'm using this branch as the base for the TMCC migration, coming up using the same principles.